docs/src/test/count_outfiles.ok

```diff
@@ -0,0 +1,170 @@
+File linked successfully.
+ERROR: README.md not found in ../src/cmake
+File linked successfully.
+ERROR: README.md not found in ../src/dbSta
+File linked successfully.
+File linked successfully.
+ERROR: README.md not found in ../src/dpo
```

Comment:
This needs fixing but can be a separate PR.  We should document improve_placement.